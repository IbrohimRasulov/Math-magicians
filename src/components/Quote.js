/* eslint-disable max-len */
const Quote = () => (
  <div className="quote">
    <fieldset>
      <legend>Quotes</legend>
      <p>
        Dear Math, please grow up and solve your own problems. I’m tired of solving them for you.
        <span>— Anonymous</span>
      </p>
      <p>
        If people do not believe that mathematics is simple, it is only because they do not realize how complicated life is.
        <span>— John von Neumann</span>
      </p>
    </fieldset>
  </div>
);

export default Quote;
