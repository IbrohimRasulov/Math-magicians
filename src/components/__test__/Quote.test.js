import renderer from 'react-test-renderer';
import Quote from '../Quote';

test('Render Quote component', () => {
  const tree = renderer.create(<Quote />);
  expect(tree).toMatchSnapshot();
});
