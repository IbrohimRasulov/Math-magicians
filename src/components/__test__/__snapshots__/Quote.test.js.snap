// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Home component 1`] = `
<div
  className="quote"
>
  <fieldset>
    <legend>
      Quotes
    </legend>
    <p>
      Dear Math, please grow up and solve your own problems. I’m tired of solving them for you.
      <span>
        — Anonymous
      </span>
    </p>
    <p>
      If people do not believe that mathematics is simple, it is only because they do not realize how complicated life is.
      <span>
        — John von Neumann
      </span>
    </p>
  </fieldset>
</div>
`;

exports[`Render Quote component 1`] = `
<div
  className="quote"
>
  <fieldset>
    <legend>
      Quotes
    </legend>
    <p>
      Dear Math, please grow up and solve your own problems. I’m tired of solving them for you.
      <span>
        — Anonymous
      </span>
    </p>
    <p>
      If people do not believe that mathematics is simple, it is only because they do not realize how complicated life is.
      <span>
        — John von Neumann
      </span>
    </p>
  </fieldset>
</div>
`;
